Dwolla.Client/DwollaClient.cs

```diff
@@ -43,15 +44,35 @@ public class DwollaClient : IDwollaClient
         public static DwollaClient Create(bool isSandbox) =>
             new DwollaClient(new RestClient(CreateHttpClient()), isSandbox);
 
-        public async Task<RestResponse<TRes>> PostAuthAsync<TReq, TRes>(Uri uri, TReq content) =>
+        public async Task<RestResponse<TRes>> PostAuthAsync<TReq, TRes>(Uri uri, TReq content) where TRes : IDwollaResponse =>
             await SendAsync<TRes>(CreatePostRequest(uri, content, new Headers(), AuthContentType));
 
-        public async Task<RestResponse<TRes>> GetAsync<TRes>(Uri uri, Headers headers) =>
+        public async Task<RestResponse<TRes>> GetAsync<TRes>(Uri uri, Headers headers) where TRes : IDwollaResponse =>
             await SendAsync<TRes>(CreateRequest(HttpMethod.Get, uri, headers));
-
-        public async Task<RestResponse<TRes>> PostAsync<TReq, TRes>(Uri uri, TReq content, Headers headers) =>
+        
+        /// <summary>
+        /// Send a post request to <paramref name="uri"/> containing <paramref name="content"/>. 
+        /// </summary>
+        /// <typeparam name="TReq">Type of the request content</typeparam>
+        /// <typeparam name="TRes">Type of the response content</typeparam>
+        /// <param name="uri">The resource to which to Post</param>
+        /// <param name="content">The content to send</param>
+        /// <param name="headers">Request headers</param>
+        /// <returns>RestResponse of deserialized content object</returns>
+        public async Task<RestResponse<TRes>> PostAsync<TReq, TRes>(Uri uri, TReq content, Headers headers) where TRes : IDwollaResponse =>

```

Comment:
Why are just these two commented? I'd say comment them all or don't comment any for consistency. My personal vote is none at all since I tried to make the names self-documenting.